src/Nethermind/Nethermind.Network/PeerPool.cs

```diff
@@ -185,23 +186,28 @@ private void StartPeerPersistenceTimer()
                 try
                 {
                     _peerPersistenceTimer.Enabled = false;
-                    RunPeerCommit();
+                    _storageCommitTask = RunPeerCommit().ContinueWith(x =>
+                    {
+                        if (x.IsFaulted && _logger.IsError)
+                        {
+                            _logger.Error($"Error during peer storage commit: {x.Exception}");
+                        }
+                        _peerPersistenceTimer.Enabled = true;
```

Comment:
not needed? it is in finally?

Reply:
Removed from finally (and added to catch) because this bit will complete after the method returns and timer thread returns to pool; so needs to be switched back on at completion.